meveo-admin-ejbs/src/main/resources/messages_fr.properties

```diff
@@ -0,0 +1,4640 @@
+AccountStatusEnum.active     = Actif
```

Comment:
Why is this file there ??

Reply:
Because ComponentResources.java requires ResourceBundle file   "messages" , so i have added 
this file this meveo project module (meveo-admin-ejbs) resource folder location. 
others side  without adding this file(messages-en.properties) into this module it gives bellow exception : 
Caused by: javax.ejb.EJBException: java.util.MissingResourceException: Can't find bundle for base name messages, locale en.

Reply:
I have copy this file(message.properties ) into two location.So,two files contains same property and it makes duplicate content.
		meveo-admin-ejbs ------->message.properties
		meveo-admin-web   ------->message.properties
		For avoid duplicate issue , i think need to update this file as purpose of meveo-admin , meveo-web. 
		Due to my meveo knowledge gap , i can not update this file.